### DISCUSSION TITLE
could not run with this configuration

### DISCUSSION THREAD

**Discussion by ruanwenfeng** [eclipse-theia/theia#10051] [Q&A] [answered]
I just want to run without any extension package except core.

![image](https://user-images.githubusercontent.com/27943586/132087442-d0797da4-88da-4d40-9900-8a04cdb4cd15.png)

**Comment by msujew** [comment, marked as answer]
Hi @ruanwenfeng, thank you for your question.

This is a known issue, but as just running the core of Theia offers basically no features (no code editor, file explorer, scm, terminals, etc), it's not something we are heavily interested in supporting. I would recommend you to install these Theia dependencies to run a minimal Theia app:

```json
"dependencies": {
    "@theia/console": "1.17.0",
    "@theia/core": "1.17.0",
    "@theia/editor": "1.17.0",
    "@theia/editor-preview": "1.17.0",
    "@theia/file-search": "1.17.0",
    "@theia/filesystem": "1.17.0",
    "@theia/markers": "1.17.0",
    "@theia/messages": "1.17.0",
    "@theia/monaco": "1.17.0",
    "@theia/navigator": "1.17.0",
    "@theia/preferences": "1.17.0",
    "@theia/process": "1.17.0",
    "@theia/search-in-workspace": "1.17.0",
    "@theia/terminal": "1.17.0",
    "@theia/workspace": "1.17.0"
}
```

**Reply by ruanwenfeng** [reply]
Thanks for your answer.   I know what to do next.
